test/test_gax.py

```diff
@@ -32,11 +32,18 @@
 
 from __future__ import absolute_import
 
+from concurrent import futures
+
+import mock
 import unittest2
 
+from fixture_pb2 import Simple
```

Comment:
It's test data: https://github.com/googleapis/gax-python/blob/master/test/fixture.proto